src/main/java/uk/gov/service/notify/NotificationClient.java

```diff
@@ -65,6 +77,18 @@ public NotificationResponse sendSms(String templateId, String to, HashMap<String
         return postRequest("sms", templateId, to, personalisation);
```

Comment:
probably worth changing the javadoc of this one to remove the word "Optional" from `* @param personalisation Optional HashMap representing the placeholders...`